src/datreant/metadata.py

```diff
@@ -1018,6 +1018,30 @@ def values(self, scope='all'):
         keys = self.keys(scope=scope)
         return self[keys]
 
+    def items(self, scope='all'):
+        """Get category (key, value) tuples for all Treants in collection.
+
+        Parameters
+        ----------
+        scope : {'all', 'any'}
+            Keys to return. 'all' will return tuples only for keys found within
+            all Treants in the collection, while 'any' will return tuples for
+            keys found within at least one Treant in the collection.
+
+        Returns
+        -------
+        items : list
```

Comment:
Shouldn’t this be a generator for py3 flavour?

Reply:
Although I don't necessarily disagree, I think nothing in `datreant` so far is a generator (save for `Tree.walk`, but that uses `scandir.walk` internally so behaves accordingly), so it would break custom to make this one work that way.

Can we open a separate issue for making these kinds of operations generators? I think if we go that route it should be systematic.